references/recaptcha-enterprise/pipeline.sh

```diff
@@ -0,0 +1,324 @@
+                "approvalType": "auto",
+                "attributes": [
+                    {
+                        "name": "access",
+                        "value": "public"
+                    }
+                ],
+                "description": "",
+                "displayName": "Recaptcha Enterprise Product",
+                "environments": [
+                    "$ENV_NAME"
+                ],
+                "proxies": [
+                    "data-proxy-v1"
+                ]
+            }
+        ],
+        "developers": [
+            {
+                "email": "janedoe@example.com",
+                "firstName": "Jane",
+                "lastName": "Doe",
+                "userName": "janedoe",
+                "attributes": []
+            }
+        ],
+        "developerApps": {
+            "janedoe@example.com": [
+                {
+                    "name": "app-recaptcha-enterprise-always1",
+                    "attributes": [
+                        {
+                            "name": "GCP_PROJECT_ID",
+                            "value": "$GCP_PROJECT_ID"
+                        },
+                        {
+                            "name": "SITE_KEY",
+                            "value": "$SITEKEY_ALWAYS_1"
+                        }
+                    ],
+                    "apiProducts": [
+                        "RecaptchaEnterprise"
+                    ],
+                    "callbackUrl": "",
+                    "scopes": []
+                },
+                {
+                    "name": "app-recaptcha-enterprise-always0",
+                    "attributes": [
+                        {
+                            "name": "GCP_PROJECT_ID",
+                            "value": "$GCP_PROJECT_ID"
+                        },
+                        {
+                            "name": "SITE_KEY",
+                            "value": "$SITEKEY_ALWAYS_0"
+                        }
+                    ],
+                    "apiProducts": [
+                        "RecaptchaEnterprise"
+                    ],
+                    "callbackUrl": "",
+                    "scopes": []
+                }
+            ]
+        }
+    }
+}
+EOF
+}
+
+#################################
+### function: set_idp_env_var ###
+#################################
+set_functional_test_env_var() {
+
+    # use timestamp (parameter) to create a unique value
+    TEST_APP_CONSUMER_KEY="xkey-$1"
+    export TEST_APP_CONSUMER_KEY
+    TEST_APP_CONSUMER_SECRET='xsecret'
+    export TEST_APP_CONSUMER_SECRET
+}
+
+######################################
+### function: set_recaptcha_config ###
+######################################
+set_recaptcha_config() {
+
+    # Copy AM policy template
+    cp "$SCRIPTPATH"/templates/AM-SetReCaptchaMock.template.xml "$SCRIPTPATH"/sf-recaptcha-enterprise-v1/sharedflowbundle/policies/AM-SetReCaptchaMock.xml
+
+    # replace reCAPTCHA tag in AM policy
```

Comment:
we also have `envsubst` in the path now if you want to simplify a bit.

Checkout this: https://github.com/apigee/devrel/blob/0920292fd1ae27391836fb3dff60343ef04aa882/references/auth-schemes/pipeline.sh#L33

Reply:
Nice !